utils/constants.py

```diff
@@ -37,7 +38,9 @@ class DataType(ValueEnum):
     """Swagger data types that we expect to deal with. Uses camelcase in values to match."""
```

Comment:
Nit: I think this comment needs updating since not all the fields are camelCase anymore

Reply:
Updated